portal/src/components/debug/DebugSearchableDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import { IDropdownOption } from "@fluentui/react";
+import React, { useCallback, useMemo, useState } from "react";
+import { useDebounced } from "../../hook/useDebounced";
+import { SearchableDropdown } from "../common/SearchableDropdown";
+
+export function DebugSearchableDropdown(): React.ReactElement {
+  const [searchKeyword, setSearchKeyword] = useState("");
+  const [selectedKey, setSelectedKey] = useState<string | null>(null);
+  const allOptions = useMemo((): IDropdownOption[] => {
+    return new Array(50).fill("").map<IDropdownOption>((_, idx) => {
+      return {
+        key: `${idx}`,
+        text: `This is option ${idx + 1}`,
+      };
+    });
+  }, []);
+
+  const [deboundedKeyword, isDebouncing] = useDebounced(searchKeyword, 300);
+
+  const options = useMemo(() => {
+    return allOptions.filter((option) =>
+      option.text.includes(deboundedKeyword.trim())
+    );
+  }, [allOptions, deboundedKeyword]);
+
+  const onChange = useCallback((_: unknown, option?: IDropdownOption) => {
+    if (!option) {
+      setSelectedKey(null);
+      return;
+    }
+    if (option.selected || option.selected === undefined) {
+      setSelectedKey(option.key as string);
+    } else {
+      setSelectedKey(null);
+    }
```

Comment:
Do we need this check? Seems like the option passed to onChange is the option selected.

Reply:
@Ngkaokis The dropdown has multi-select mode. In multi-select mode, we need to check `selected` to know if it is a selection of deselection.